api2-war/src/main/java/eu/europeana/api2/v2/web/controller/ContentReuseFrameworkController.java

```diff
@@ -29,7 +53,12 @@
     @Resource
     private SearchService searchService;
 
-  
+    @Resource(name = "corelib_web_contentReuseFrameworkService")
```

Comment:
removed qualifier. 

Why is it a good idea to have that (IMHO) :
the idea of having annotations in code for autowiring dependencies like this makes the design of the app brittle => bad design (the Spring authors discourage autowiring except for very small projects); having a qualifier reduces the britleness of the design a bit.